week-3/C-array-every/exercise.js

```diff
@@ -5,7 +5,15 @@
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 var group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-var groupIsOnlyStudents; // complete this statement
+function Check(names) {
+  if (students == group) {
```

Comment:
This isn't quite right. Using `==` to compare 2 arrays doesn't work. And here we don't want to compare the whole `group` array against `students`. We want to compare each individual element in the `group` array instead. How can we fix that?